### DISCUSSION TITLE
offsite-disaster-recovery-for-unraid-with-rclone

### DISCUSSION THREAD

**Discussion by bowencool** [bowencool/blog#43] [Announcements]
https://blog.bowen.cool/zh/posts/offsite-disaster-recovery-for-unraid-with-rclone

**Comment by geekbb** [comment]
谢谢，非常详细！

**Comment by bowencool** [comment]
# 更新
flash backup 也很简单，就是把 /boot 目录压缩就行了：
``` bash
tar -czvf /tmp/`hostname`_flash.tgz --exclude 'previous*' --exclude "System Volume Information" --exclude 'logs*' /boot
```

**Comment by seekdoor** [comment]
看完这个，我有种把百度云盘换成阿里云盘的冲动了。百度云盘没阿里云盘这么开放

**Reply by bowencool** [reply]
国内云盘其实都差不多的，可以看下[第二篇文章](https://blog.bowen.cool/zh/posts/how-to-encrypt-backup-your-data-on-your-nas#%E5%82%A8%E5%AD%98%E7%BB%88%E7%82%B9%E7%9A%84%E9%80%89%E6%8B%A9)，你可以试试用 alist 将百度云盘转换成 WebDAV 协议
